static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js

```diff
@@ -42,7 +42,7 @@ define([
                         contributionUrl: variant.options.contributeURL,
                         componentName: variant.options.componentName,
                         quoteSvg: quoteSvg.markup,
-                        p1: '&hellip; we have a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help, especially during this UK election.',
+                        p1: '&hellip; we have a small favour to ask. More people are reading the Guardian than ever but advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help, especially during this UK election.',
```

Comment:
not stricly PR feedback, but would it be better to say "More people than ever are reading the Guardian" rather than "More people are reading the Guardian than ever" ?